src/h5web/vis-packs/core/heatmap/HeatmapMesh.tsx

```diff
@@ -25,20 +26,7 @@ interface Props {
   alphaDomain?: Domain;
 }
 
-const SCALE_FUNC: Record<ScaleType, (val: number) => number> = {
-  [ScaleType.Linear]: (val) => val,
-  [ScaleType.Log]: Math.log10,
-  [ScaleType.SymLog]: (val) => {
-    return Math.sign(val) * Math.log10(1 + Math.abs(val));
-  },
-};
-
 const CMAP_SIZE = 256;
-const CMAP_NORM: Record<ScaleType, number> = {
-  [ScaleType.Linear]: 0,
-  [ScaleType.Log]: 1,
-  [ScaleType.SymLog]: 2,
-};
```

Comment:
This was replaced by an explicit implementation of the scale shader in the h5web scale entry.